prog/dftb+/lib_dftb/CMakeLists.txt

```diff
@@ -3,6 +3,7 @@ set(curdir "lib_dftb")
 set(sources-fpp
   ${curdir}/chargeconstr.F90
   ${curdir}/charges.F90
+  ${curdir}/clscorr.F90
```

Comment:
What about `classicalcorr.F90`? Still within the 15 char limit and probably more informative. :wink: